control-plane-operator/controllers/hostedcontrolplane/cno/clusternetworkoperator.go

```diff
@@ -0,0 +1,234 @@
+	NetworkCheckTarget           string
+	CloudNetworkConfigController string
+}
+
+type Params struct {
+	ReleaseVersion          string
+	AvailabilityProberImage string
+	APIServerAddress        string
+	APIServerPort           int32
+	TokenAudience           string
+	Images                  Images
+	OwnerRef                config.OwnerRef
+	DeploymentConfig        config.DeploymentConfig
+}
+
+func NewParams(hcp *hyperv1.HostedControlPlane, version string, images map[string]string, setDefaultSecurityContext bool) Params {
+
+	// TODO: Remove
+	images["cluster-network-operator"] = "quay.io/pdiak/cno:dev"
+
+	p := Params{
+		Images: Images{
+			NetworkOperator:              images["cluster-network-operator"],
+			SDN:                          images["sdn"],
+			KubeProxy:                    images["kube-proxy"],
+			KubeRBACProxy:                images["kube-rbac-proxy"],
+			Multus:                       images["multus-cni"],
+			MultusAdmissionController:    images["multus-admission-controller"],
+			CNIPlugins:                   images["container-networking-plugins"],
+			BondCNIPlugin:                images["network-interface-bond-cni"],
+			WhereaboutsCNI:               images["multus-whereabouts-ipam-cni"],
+			RouteOverrideCNI:             images["multus-route-override-cni"],
+			MultusNetworkPolicy:          images["multus-networkpolicy"],
+			OVN:                          "quay.io/zshi/ovn-daemonset:topology-version", // TODO: use images["ovn-kubernetes"]
+			EgressRouterCNI:              images["egress-router-cni"],
+			KuryrDaemon:                  images["kuryr-cni"],
+			KuryrController:              images["kuryr-controller"],
+			NetworkMetricsDaemon:         images["network-metrics-daemon"],
+			NetworkCheckSource:           images["cluster-network-operator"],
+			NetworkCheckTarget:           images["cluster-network-operator"],
+			CloudNetworkConfigController: images["cloud-network-config-controller"],
+		},
+		ReleaseVersion:          version,
+		AvailabilityProberImage: images[util.AvailabilityProberImageName],
+		OwnerRef:                config.OwnerRefFrom(hcp),
+	}
+
+	p.DeploymentConfig.Scheduling.PriorityClass = config.DefaultPriorityClass
+	p.DeploymentConfig.SetColocation(hcp)
+	p.DeploymentConfig.SetRestartAnnotation(hcp.ObjectMeta)
+	p.DeploymentConfig.SetReleaseImageAnnotation(hcp.Spec.ReleaseImage)
+	p.DeploymentConfig.SetControlPlaneIsolation(hcp)
+	p.DeploymentConfig.Replicas = 1
+	p.DeploymentConfig.SetDefaultSecurityContext = setDefaultSecurityContext
+	p.APIServerAddress = hcp.Status.ControlPlaneEndpoint.Host
+	p.APIServerPort = hcp.Status.ControlPlaneEndpoint.Port
+	p.TokenAudience = hcp.Spec.IssuerURL
+
+	return p
+}
+
+func ReconcileServiceAccount(sa *corev1.ServiceAccount, ownerRef config.OwnerRef) error {
+	ownerRef.ApplyTo(sa)
+	return nil
+}
+
+func ReconcileRole(role *rbacv1.Role, ownerRef config.OwnerRef) error {
+	ownerRef.ApplyTo(role)
+	role.Rules = []rbacv1.PolicyRule{
```

Comment:
Suggest adding a comment here that indicates these are for running ovn-k master in the management cluster.